### ISSUE TITLE
Nextflow

### ISSUE THREAD

**Issue by dsajdak** [ubccr/software-layer#338]
Nextflow is used by many bioinformatics labs and I think it would be useful to have installed in our repository
https://github.com/easybuilders/easybuild-easyconfigs/tree/develop/easybuild/easyconfigs/n/Nextflow

**Comment by tonykew** [comment]
Nextflow version 23.10.0 has been built, and is going through internal testing.
I will update this request once Nextflow has been published.

**Comment by tonykew** [comment]
Nextflow version 23.10.0 has been published to the ccrsoft/2023.01 software release.

```
login1$ module spider nextflow
----------------------------------------------------------------------------
  nextflow: nextflow/23.10.0
----------------------------------------------------------------------------
    Description:
      Nextflow is a reactive workflow framework and a programming DSL that
      eases writing computational pipelines with complex data


    This module can be loaded directly: module load nextflow/23.10.0
[...]
login1$ module load nextflow/23.10.0
login1$ nextflow help
Usage: nextflow [options] COMMAND [arg...]

Options:
  -C
     Use the specified configuration file(s) overriding any defaults
  -D
     Set JVM properties
  -bg
     Execute nextflow in background
  -c, -config
     Add the specified file to configuration set
  -config-ignore-includes
     Disable the parsing of config includes
  -h
     Print this help
  -log
     Set nextflow log file path
  -q, -quiet
     Do not print information messages
  -remote-debug
     Enable JVM interactive remote debugging (experimental)
  -syslog
     Send logs to syslog server (eg. localhost:514)
  -trace
     Enable trace level logging for the specified package name - multiple packages can be provided separating them with a comma e.g. '-trace nextflow,io.seqera'
  -v, -version
     Print the program version

Commands:
  clean         Clean up project cache and work directories
  clone         Clone a project into a folder
  config        Print a project configuration
  console       Launch Nextflow interactive console
  drop          Delete the local copy of a project
  help          Print the usage help for a command
  info          Print project and system runtime information
  inspect       Inspect process settings in a pipeline project
  kuberun       Execute a workflow in a Kubernetes cluster (experimental)
  list          List all downloaded projects
  log           Print executions log and runtime info
  plugin        Execute plugin-specific commands
  plugins       Execute plugin-specific commands
  pull          Download or update a project
  run           Execute a pipeline project
  secrets       Manage pipeline secrets (preview)
  self-update   Update nextflow runtime to the latest available version
  view          View project script file(s)

login1$ 

```
